src/cdk/v2/destinations/intercom/utils.test.js

```diff
@@ -0,0 +1,350 @@
+const {
```

Comment:
Can we add test cases for `createOrUpdateCompany` and `searchContact` util functions

Reply:
addressed